README.md

```diff
@@ -788,6 +855,25 @@ RSpec.configure do |config|
 end
 ```
 
+### Elasticsearch client options
+
+All connection options, except the `:prefix`, are passed to the `Elasticseach::Client.new` ([chewy/lib/chewy.rb](https://github.com/toptal/chewy/blob/f5bad9f83c21416ac10590f6f34009c645062e89/lib/chewy.rb#L153-L160)):
+
+```ruby	
+# Main elasticsearch-ruby client instance
+#
+def client
+  Thread.current[:chewy_client] ||= begin
+    client_configuration = configuration.deep_dup
+    client_configuration.delete(:prefix) # used by Chewy, not relevant to Elasticsearch::Client
+    block = client_configuration[:transport_options].try(:delete, :proc)
+    ::Elasticsearch::Client.new(client_configuration, &block)
+  end
+end
+```
+
+Elasticsearch side documentation: https://rubydoc.info/gems/elasticsearch-transport#setting-hosts
```

Comment:
Here's the (relevant) Elasticsearch documentation on the subject (or something like this).